pkgs/development/libraries/stfl/default.nix

```diff
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
     DESTDIR=$out prefix=\"\" make install
 
     # some programs rely on libstfl.so.0 to be present, so link it
-    ln -s $out/lib/libstfl.so.0.22 $out/lib/libstfl.so.0
+    ln -s $out/lib/libstfl.so.0.24 $out/lib/libstfl.so.0
```

Comment:
was this a manual fix or...?

Reply:
That was done automatically because the version umber happens to be 0.22 and 0.24... I'm not sure if that's right.

Reply:
Checked that this is correct